src/graph/planner/ngql/PathPlanner.cpp

```diff
@@ -355,23 +356,24 @@ PlanNode* PathPlanner::buildEdgePlan(PlanNode* dep, const std::string& input) {
   // col 0 of the project->output() is [edge...]
   auto* unwindExpr = ColumnExpression::make(pool, 0);
   auto* unwind = Unwind::make(qctx, project, unwindExpr);
+  unwind->setFromPipe(true);
   unwind->setColNames({"edges"});
 
   // extract src from edge
   auto srcArgs = ArgumentList::make(pool);
-  srcArgs->addArgument(ColumnExpression::make(pool, 1));
```

Comment:
Maybe we could use constant variable instead of number literal.

Reply:
Or property name directly. And optimize hash map lookup in inner implementation.

Reply:
there is a detailed note above, 0 for column 0

Reply:
And user could turn off `properties pruning`, so the column index number will lead to mismatched property.

Reply:
this is nothing to do with properties pruning, it is just an optimization of find path, no need to output additional data